### ISSUE TITLE
Add Document from localGPT UI is not working 

### ISSUE THREAD

**Issue by vermashubhangi** [PromtEngineer/localGPT#675]
(https://github.com/PromtEngineer/localGPT/assets/46860801/76381503-b253-4223-b3ae-eb30b5fae67d)

**Comment by AIGuyBNE** [comment]
Yes it seems that localGPTUI.py is just a straightforward chat interface with no menu or options to chat directly with your documents

**Comment by AIGuyBNE** [comment]
![Screenshot from 2023-12-10 11-45-07](https://github.com/PromtEngineer/localGPT/assets/153247194/4d6a5b08-8986-411a-bf2b-ee60bfbde71b)

The '/localGPTUI/templates/home.html' file when loaded directly has all the options you would expect to see. All the menus aren't being rendered

**Comment by TechInnovate01** [comment]
> ![Screenshot from 2023-12-10 11-45-07](https://private-user-images.githubusercontent.com/153247194/289332892-4d6a5b08-8986-411a-bf2b-ee60bfbde71b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDQ0NDk1MTcsIm5iZiI6MTcwNDQ0OTIxNywicGF0aCI6Ii8xNTMyNDcxOTQvMjg5MzMyODkyLTRkNmE1YjA4LTg5ODYtNDExYS1iZjJiLWVlNjBiZmJkZTcxYi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMTA1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDEwNVQxMDA2NTdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT01YjQ2Nzk0NjQ5ZjUzNzJiNzRkNGZlZjFhMWJiZDRjMDVlOGE2Y2NlMDJjZTRhODc2NDg2Y2JhNzY1ZTIyZjlkJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.CVrUDMOfUpl3_-CBd3u-T3lgVZ0WClb5S2tWrLkKVas)
> 
> The '/localGPTUI/templates/home.html' file when loaded directly has all the options you would expect to see. All the menus aren't being rendered

you have to run this through command prompt itself.
refer the section name "Run the Graphical User Interface" in readme which will help you to use UI.
Hope this help.

**Comment by NitkarshChourasia** [comment]
@vermashubhangi @AIGuyBNE 

# WAS THE ISSUE SOLVED?
Because when I try to run the UI, it is just prompting.
It is like an actual Openai - ChatGPT 3.5 interface.
It wasn't having an option to upload documents or anything.
So, I thought I have to ingest the documents by ```python ingest.py --device_type gpu``` then after running it. I can access the vector database through the UI, simply.
With all the Blackbox things happening behind the scenes.

What I am thinking is right?
Thank you.
